stylis.js

```diff
@@ -958,9 +958,13 @@
 					out = webkit + out.replace(gradientptn, webkit+'$1') + out
 					break
 				}
-				// filter
+				// fill/filter, f, i, l
 				case 951: {
-					out = webkit + out + out
+          // filter
+          if (out.charCodeAt(3) === 116) {
+            console.log(out, out.charCodeAt(5));
```

Comment:
oops :)

Reply:
Haha sorry, should be fixed

Reply:
Oh god my editor's Prettier has gone crazy on all the linebreaks, let me fix that quickly

Reply:
Atom _really_ likes messing with whitespace. All sorted now 👍